src/ngx_http_lua_headers_out.c

```diff
@@ -106,6 +106,12 @@ static ngx_http_lua_set_header_t  ngx_http_lua_set_handlers[] = {
                  offsetof(ngx_http_headers_out_t, cache_control),
                  ngx_http_set_builtin_multi_header },
 
+#if defined(nginx_version) && nginx_version >= 1013010
```

Comment:
This change was introduced by Nginx 1.13.9, and not 1.13.10 (https://github.com/nginx/nginx/commit/8590d9d615d0b83cf076421d1e18c64ca66a1a44).

Reply:
Oh, my bad, this was correct with `1013010`.

Reply:
Actually, no, here is such an example in which we use the Nginx _release_ introducing a symbol, and _not_ the version bump commit: [this commit](https://github.com/nginx/nginx/commit/afad21917584e9b452ba33ce3485edde5615b859), introducing `ngx_parent` was added when `nginx_version` was bumped to `1.13.8`, but _before_ the `1.13.8` release. Yet, in our FFI API, we check for `nginx_version >= 1013008` ([source](https://github.com/openresty/lua-nginx-module/blob/master/src/ngx_http_lua_worker.c#L141)).

I think this check should be `nginx_version >= 1013009` after all.

Reply:
OK, gotcha :)